assignment/week4/src/components/mypage/Header.tsx

```diff
@@ -0,0 +1,116 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { Link } from "react-router";
+import { AnimatePresence } from "framer-motion";
```

Comment:
```suggestion
import { AnimatePresence } from "motion/react";
```
framer motion은 motion으로 독립되었어요! 

https://motion.dev/blog/framer-motion-is-now-independent-introducing-motion